app/javascript/components/applications/application_form/DemographicInfoSection.js

```diff
@@ -61,8 +54,7 @@ const DemographicInfoSection = ({ values: { demographics } }) => {
           <InputField
             fieldName='demographics.sexual_orientation_other'
             label='Sexual Orientation (if not listed)'
-            maxLength={maxLengthMap['sexual_orientation_other']}
-            validation={validateNotListed('Sexual Orientation is required', 'sexual_orientation')} />
+            maxLength={maxLengthMap['sexual_orientation_other']} />
```

Comment:
@LoganArnett looks like some sort of merge conflict here undoing your gender validation changes?

Reply:
This is actually in relation to how the validation was working and how we are not using the direct react-final-form values and handlers connected so I put the validation at the form level instead of the field level so that the tests pass on blur or change

Reply:
ok I will re-test this stuff